### PR TITLE
Add enviornment variables screenshots

### PR DESCRIPTION
**Pull request by chrieke**
### Scope of the PR

Realized from the error dashboard that some people seem to incorrectly use the environment variables. Adding this screenshot example to the import & export data blocks.
image

### Checklist

- [x] Checked spelling and grammar
- [x] Checked the output of `make html` to ensure new content displays correctly

### CODE REVIEW

File: source/up42-blocks/data/geotiff-custom.rst [hunk 1]
```diff
@@ -22,10 +22,13 @@ The block outputs the scene data and an automatically created `data.json` file w
```
Comment (perusio) [inline source/up42-blocks/data/geotiff-custom.rst:22]: Without splitting the credentials passthrough from OA from the env vars thing in itself we have to remove this reference **everywhere**.

File: source/up42-blocks/data/dimap-custom.rst [hunk 2]
```diff
@@ -23,10 +23,13 @@ The block outputs the scene data and an automatically created `data.json` file w
 
     In order to access the bucket, the access credentials need to be provided via :ref:`UP42 environment variables <environments-credentials-pass-through-tutorial>`.
```
Comment (perusio) [inline source/up42-blocks/data/dimap-custom.rst:24]: Ditto (see below).

File: source/up42-blocks/processing/export-data-raster.rst [hunk 3]
```diff
@@ -19,10 +19,13 @@ The user can optionally provide a `prefix` for an output directory within the bu
 
     In order to access the bucket, the access credentials need to be provided via :ref:`UP42 environment variables <environments-credentials-pass-through-tutorial>`.
```
Comment (perusio) [inline source/up42-blocks/processing/export-data-raster.rst:20]: Re-ditto

File: source/up42-blocks/processing/export-data-vector.rst [hunk 4]
```diff
@@ -19,10 +19,13 @@ The user can optionally provide a `prefix` for an output directory within the bu
 
     In order to access the bucket, the access credentials need to be provided via :ref:`UP42 environment variables <environments-credentials-pass-through-tutorial>`.
```
Comment (perusio) [inline source/up42-blocks/processing/export-data-vector.rst:20]: Re-Re-ditto

**Review by perusio — COMMENTED** [review]
I say we hold this off until the OA passthrough is split from the env vars thing in itself.

**Comment by chrieke** [comment]
> I say we hold this off until the OA passthrough is split from the env vars thing in itself.

We can additionally add a link to the finished chapter when split off, but would like to merge this anyway. The blocks are already public and we got some errors with the use of the env variables, this will give a bit more context. Splitting of the chapter should be high prio regardless, is there a Jira docs board etc?

**Comment by chrieke** [comment]
If no further objection merging this tomorrow.

**Comment by perusio** [comment]
> If no further objection merging this tomorrow.

the objection is not from but from @seanwiid. If he agrees we go public with referring that page: yes. Then we need to add it to the index, because is not even indexed by bots. It has a `no-follow` meta tag.

**Comment by chrieke** [comment]
@perusio Let's talk tomorrow, I think there is a missunderstanding here. This PR adds a screenshot to the environments use which is public and the blocks are public. This does not refer to OA passthrough in any way.

**Comment by chrieke** [comment]
@perusio Now I understand the issue. The reference is in the already public chapter, not in this PR. Removing the link now, then this can be merged.

**Review by perusio — APPROVED** [review]
LGTM.

**Pull request merged**